src/components/views/right_panel/RoomSummaryCard.tsx

```diff
@@ -76,7 +76,7 @@ const Button: React.FC<IButtonProps> = ({ children, className, onClick }) => {
 };
 
 export const useWidgets = (room: Room) => {
-    const [apps, setApps] = useState<IApp[]>(WidgetStore.instance.getApps(room.roomId));
+    const [apps, setApps] = useState<IApp[]>(() => WidgetStore.instance.getApps(room.roomId));
```

Comment:
Is this going to ignore the `room` changing? Why would we want that?

(am I forgetting something about this React feature again? :sweat_smile:)

Reply:
`room` changing is accounted for by the `useEffect(updateApps, [room, updateApps]);` line after this. My only reason for this change was optimization, so that `getApps` doesn't get called on every render.